changelog.md

```diff
@@ -4,10 +4,35 @@ All notable end-user facing changes are documented in this file.
 ### [vNext]
 *Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
 The idea is to track end-user facing changes as they occur.*
-- Add streaming support for types that are serialized using IExternalSerializers
-- Replace CallContext.LogicalSetData with AsyncLocal #2200
-- Providers are now constructed using Dependency Injection. The result is that all providers must have a single public constructor with either no arguments or arguments which can all be injected. The default container is used if no container is configured. [#2721](https://github.com/dotnet/orleans/pull/2721)
-- Bug fixes:
+
+### [1.5.0-beta1]
+- Breaking changes
+  - Upgrade minimum framework dependency to .NET 4.6.1 #2945
+  - Support for non-static client via ClientBuilder (although static GrainClient still works but will be removed in a future version). You can now start many clients in the same process, even if you are inside a Silo #2822.
+    There are a few differences though:
+    - Several changes to SerializationManager, mainly to make it non-static #2592
+      - When deserializing a GrainReference from storage, you might need to re-bind it to the runtime by calling `grain.BindGrainReference(grainFactory)` on it or you would get `GrainReferenceNotBoundException` when attempting to use it #2738
+      - Removed RegisterSerializerAttribute (and corresponding static Register method for registering a custom serializer). If you were using that, please use `[Serializer(typeof(TargetType))]` instead, similar to how the generated code works #2941
+  - Better serialization of `Type` values (but can cause compatibility issues if these were persisted by using the Serialziation Manager) #2952
+  - Automatically deactivate a grain if it bubbles up `InconsistentStateException` (thrown when there is an optimistic concurrency conflict when writing to storage) #2959
+  - Providers are now constructed using Dependency Injection. The result is that all providers must have a single public constructor with either no arguments or arguments which can all be injected. #2721
+  - Change default stream subscription faulting to false in EventHub and Memory stream providers, as is in other providers #2974
+  - Bug fix: Azure storage providers now throw `InconsistenStateException` instead of `StorageException` when eTags do not match #2971
+- Non-breaking improvements
+	- Grain interface versioning to enable no-downtime upgrades #2837 2837
```

Comment:
:+1: was using VSCode, and since we don't have an .editorconfig, it uses tabs automatically